### PR TITLE
Add CD Events to the landscape

### PR DESCRIPTION
**Pull request by e-backmark-ericsson**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [X] Are you only including a `repo_url` if your project is 100% open source? If so, pick the single best GitHub repository for your project, not a GitHub organization.
* [-] Does it follow the guidelines for [new entries](https://github.com/cdfoundation/cdf-landscape#new-entries)?
* [-] Have you added an SVG to `hosted_logos` and referenced it there?
* [-] Does your logo clearly state the name of the project/product and follow logo [guidelines](https://github.com/cdfoundation/cdf-landscape#logos)?
* [-] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [-] ~5 minutes after opening the pull request, the CNCF-Bot will post a link for you to preview your changes. Have you confirmed that they are correct and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Review by markyjackson-taulia — CHANGES_REQUESTED** [review]
Line 385 must have a name

**Comment by e-backmark-ericsson** [comment]
> Line 385 must have a name

Yeah. My bad. Copy/paste issue... ;)

**Comment by e-backmark-ericsson** [comment]
It also lacks logo, homepage and crunchbase. That's why it's a draft. We're waiting for the logo to be settled :)

**Comment by markyjackson-taulia** [comment]
oh interesting, my original review did not show it as a draft and that is why I reviewed, my bad if I somehow missed that.

File: landscape.yml [hunk 1]
```diff
@@ -378,6 +378,10 @@ landscape:
             repo_url: https://github.com/eiffel-community/eiffel
             logo: eiffel.svg
             crunchbase: https://www.crunchbase.com/organization/eiffel-community
+          - item:
+            name: CD Events
+            description: CD Events is an event protocol providing interoperability in CD systems
+            repo_url: https://github.com/cdevents/spec
```
Comment (oleg-nenashev) [inline landscape.yml:384]:
```suggestion
            repo_url: https://github.com/cdevents/spec
            crunchbase: https://www.crunchbase.com/organization/continuous-delivery-foundation-cdf
```

**Comment by e-backmark-ericsson** [comment]
Currently waiting for https://github.com/cdfoundation/foundation/issues/348

File: landscape.yml [hunk 2]
```diff
@@ -378,6 +378,11 @@ landscape:
...
```
Comment (m-linner-ericsson) [inline landscape.yml:384]: I guess we only have to add cd events logo now?

**Review by m-linner-ericsson — CHANGES_REQUESTED** [review]

File: landscape.yml [hunk 3]
```diff
@@ -378,6 +378,11 @@ landscape:
             repo_url: https://github.com/eiffel-community/eiffel
             logo: eiffel.svg
             crunchbase: https://www.crunchbase.com/organization/eiffel-community
+          - item:
+            name: CD Events
+            description: CD Events is an event protocol providing interoperability in CD systems
```
Comment (m-linner-ericsson) [inline landscape.yml:383]: We need to align to other places where spell out the name. Is this the correct form?
```suggestion
            name: CDEvents
            description: CDEvents is an event protocol providing interoperability in CD systems
```

**Review by m-linner-ericsson — CHANGES_REQUESTED** [review]

File: landscape.yml (hunk 2 above)
Comment (e-backmark-ericsson) [reply landscape.yml:384]: Yes. Or rather the stacked version, as specified here: https://github.com/cdfoundation/cdf-landscape#logos

File: landscape.yml (hunk 3 above)
Comment (e-backmark-ericsson) [reply landscape.yml:383]: Correct, I will add that in my coming update

**Comment by e-backmark-ericsson** [comment]
Just lacking the homepage_url right now. Working on it.

**Comment by MarckK** [comment]
LGTM!

**Pull request merged**